web/tests/people.test.js

```diff
@@ -1077,6 +1077,15 @@ test_people("get_mention_syntax", () => {
     assert.equal(people.get_mention_syntax("all", 1203), "@**all|1203**");
 });
 
+test_people("is_wildcard_mention", () => {
+    assert.ok(!people.is_wildcard_mention("@**Stephen King**"));
+    assert.ok(!people.is_wildcard_mention("@_**Stephen King**"));
+    assert.ok(people.is_wildcard_mention("@**all**"));
+    assert.ok(people.is_wildcard_mention("@_**all**"));
+    assert.ok(people.is_wildcard_mention("@**everyone**"));
+    assert.ok(people.is_wildcard_mention("@**stream**"));
+})
```

Comment:
This test is problematic, in that it doesn't exercise the case without an `@` at all.

Reply:
OK.